tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/new_message.py

```diff
@@ -29,8 +29,9 @@ def __init__(self, marionette):
         Base.__init__(self, marionette)
         Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
         self.apps.switch_to_displayed_app()
-        section = self.marionette.find_element(*self._thread_messages_locator)
-        Wait(self.marionette).until(lambda m: section.location['x'] == 0)
+        element = Wait(self.marionette).until(
+            expected.element_present(*self._panel_conversationview_locator))
+        Wait(self.marionette).until(lambda m: element.rect['x'] == 0 and element.is_displayed())
```

Comment:
Location is deprecated, see: http://mxr.mozilla.org/mozilla-central/source/testing/marionette/driver/marionette_driver/marionette.py#140

See https://bugzilla.mozilla.org/show_bug.cgi?id=1155746 for replacing it with rect (internally, location is already using rect.